omnibus/resources/dogstatsd/msi/localbuild/localization-en-us.wxl

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="utf-8"?>
+<WixLocalization Culture="en-us" xmlns="http://schemas.microsoft.com/wix/2006/localization">
+  <String Id="LANG">1033</String>
+  <String Id="ProductName" Overridable="no" Localizable="no">Datadog Dogstatsd</String>
+  <String Id="ManufacturerName" Overridable="no" Localizable="no">Datadog, Inc.</String>
+  <String Id="CompanyName" Overridable="no" Localizable="no">Datadog, Inc.</String>
+  <String Id="CompanyNameShort" Overridable="no" Localizable="no">Datadog</String>
+  <String Id="InstallerComment">Copyright 2015 [CompanyName]</String>
+
+  <!-- these strings are for the main agent service install -->
+  <String Id="ServiceDisplayName">[ProductName]</String>
+  <String Id="ServiceDescription">Send metrics to [ShortCompanyName]</String>
+  <!-- these strings are for the apm agent service install -->
+  <String Id="ServiceApmDisplayName">[ShortCompanyName] Trace Agent</String>
+  <String Id="ServiceApmDescription">Send tracing metrics to [ShortCompanyName]</String>
+
+  <!-- these strings are for the process agent service install -->
+  <String Id="ServiceProcessDisplayName">[ShortCompanyName] Process Agent</String>
+  <String Id="ServiceProcessDescription">Send process metrics to [ShortCompanyName]</String>
+  
+
+  <String Id="WelcomeDlgTitle">{\WixUI_Font_Bigger}Welcome to the [ProductName] Setup Wizard</String>
+  <String Id="WelcomeDlgDescription">{\WixUI_Font_Normal}The Setup Wizard will install [ProductName] on your computer.  Click Next to continue or Cancel to exit the Setup Wizard</String>
+
+  <String Id="LicenseAgreementDlgTitle">{\WixUI_Font_Title_White}End-User License Agreement</String>
+  <String Id="LicenseAgreementDlgDescription">{\WixUI_Font_Normal_White}Please read the following license agreement carefully</String>
+
+  <String Id="InstallDirDlgTitle">{\WixUI_Font_Title_White}Destination Folder</String>
+  <String Id="InstallDirDlgDescription">{\WixUI_Font_Normal_White}Click Next to install to the default folder or click Change to choose another.</String>
+
+  <String Id="ProgressDlgTitleInstalling">{\WixUI_Font_Title_White}Installing [ProductName]</String>
+
+  <String Id="VerifyReadyDlgInstallTitle">{\WixUI_Font_Title_White}Ready to install [ProductName]</String>
+
+  <String Id="FeatureMainName">Dogstatsd</String>
+  <String Id="LaunchAgentManager"> Launch Datadog Agent Manager</String>
```

Comment:
Do we package the Datadog Agent Manager GUI in a dogstatsd-only build? I'd guess not, since we don't package the main Agent.

Reply:
No, but having the strings in the localization file won't hurt. They won't even end up in the final msi.